src/hotspot/share/gc/serial/serialHeap.cpp

```diff
@@ -92,26 +90,24 @@ SerialHeap* SerialHeap::heap() {
 }
 
 SerialHeap::SerialHeap() :
-    CollectedHeap(),
-    _young_gen(nullptr),
-    _old_gen(nullptr),
-    _rem_set(nullptr),
-    _soft_ref_policy(),
-    _gc_policy_counters(new GCPolicyCounters("Copy:MSC", 2, 2)),
-    _incremental_collection_failed(false),
-    _full_collections_completed(0),
-    _young_manager(nullptr),
-    _old_manager(nullptr),
-
-    _eden_pool(nullptr),
-    _survivor_pool(nullptr),
-    _old_pool(nullptr) {
+  CollectedHeap(),
+  _young_gen(nullptr),
+  _old_gen(nullptr),
+  _rem_set(nullptr),
+  _soft_ref_policy(),
+  _gc_policy_counters(new GCPolicyCounters("Copy:MSC", 2, 2)),
+  _incremental_collection_failed(false),
+  _full_collections_completed(0),
+  _young_manager(nullptr),
+  _old_manager(nullptr),
+  _eden_pool(nullptr),
+  _survivor_pool(nullptr),
+  _old_pool(nullptr) {
```

Comment:
Don't make this change to the indentation of the mem-initializer-list.  The existing indentation is one of
several styles used in GC code, including throughtout Z.  The proposed new indentation is one which is
also probably still used in older code.  But some time ago there was discussion amongst the Oracle GC
team about constructor formatting and which of several we might prefer.  We didn't reach a consensus
at the time, but this one was unanimously, and strongly, disliked.  The problem is that with the indentation
of the mem-initializer-list being the same as the body, and the opening brace of the body cuddled with
the end of the mem-initializer-list, it is hard to spot where the mem-initializer-list ends and the body
begins.